scout/server/config.py

```diff
@@ -87,6 +87,22 @@
 #    "loqus_api" : {"api_url": "http://127.0.0.1:9000"},
 # }
 
+# Define here the VCF keys you want to be parsed for eventual custom archived LoqusDB instances.
+# Comment out if no archived LoqusDB instance is annotated in your VCF files
+LOQUSDB_ARCHIVE_VCF_INFO_FIELDS = {
+    # single keys (map VCF INFO key → target field in parsed_variant)
+    "Obs": {"field": "local_obs_old", "type": int},
+    "Hom": {"field": "local_obs_hom_old", "type": int},
+    "Frq": {"field": "local_obs_old_freq", "type": float},
+    "clinical_genomics_loqusObs": {"field": "local_obs_old", "type": int},
+    "clinical_genomics_loqusFrq": {"field": "local_obs_old_freq", "type": float},
+    "clin_obs": {"field": "local_obs_old", "type": int},
+    # prefixes → auto-expand into *_Obs, *_Hom, *_Frq
```

Comment:
Is this too complex to have in the config file? Would it be better with a long list of single keys instead?

Reply:
I like the idea of removing hard coded keys as much as possible. Perhaps we could try to just have a list (no matter how long) of keys (containing prefixes and suffixes). That would be very clear